DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -612,6 +613,21 @@ function FixFlagTriggers()
     }
 }
 
+function FixBeamLaserTriggers()
+{
+#ifdef fixes
+    local #var(prefix)BeamTrigger bt;
+    local #var(prefix)LaserTrigger lt;
+
+    foreach AllActors(class'#var(prefix)BeamTrigger',bt){
+        bt.TriggerType=TT_AnyProximity;
```

Comment:
any reason why we don't just do this in the BeginPlay of the new injects classes?